dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -731,7 +817,22 @@ const hoverHeadlineByline = (format: ArticleFormat): string => {
 
 const textRichLink: (format: ArticleFormat) => string = (format) => {
 	if (format) {
-		return pillarPalette[format.theme].main;
+		switch (format.theme) {
```

Comment:
These changes aren't scoped only to deadblogs (?) - is this ok?

Reply:
In RichLink we pass the format of the article we're linking to, e.g. when we're linking to an article, the design will be Standard and not Deadblog. So if we add a case for deadblogs, it won't be reached. I was ambivalent as to what to do but decided to make the change as Culture and Lifestyle richLinks are not accessible atm. Thanks for pointing this out, I will double check with @HarryFischer!

Reply:
It is okay to make this change 👍